utils/src/main/kotlin/ArchiveUtils.kt

```diff
@@ -182,8 +182,9 @@ fun File.unpack7Zip(targetDirectory: File) {
  * Pack the file into a ZIP [targetFile] using [Deflater.BEST_COMPRESSION]. If the file is a directory its content is
  * recursively added to the archive. Only regular files are added, e.g. symbolic links or directories are skipped. If
  * a [prefix] is specified, it is added to the file names in the ZIP file.
+ * If not all files shall be added to the archive a [filter] can be provided.
```

Comment:
Is the renaming of `FileSaver` really supposed to be in this commit?

Reply:
No, fixed.